scipy/sparse/linalg/dsolve/linsolve.py

```diff
@@ -465,11 +466,15 @@ def spsolve_triangular(A, b, lower=True, overwrite_A=False, overwrite_b=False):
         Enabling gives a performance gain. Default is False.
         If `overwrite_b` is True, it should be ensured that
         `b` has an appropriate dtype to be able to store the result.
+    unit_diagonal : bool, optional
+        If True, diagonal elements of `a` are assumed to be 1 and will not be
+        referenced.
+        .. versionadded:: 1.4.0
```

Comment:
Empty line before rst directives (doesn't render correctly otherwise https://14642-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.sparse.linalg.spsolve_triangular.html#scipy.sparse.linalg.spsolve_triangular)

Reply:
New line added.